packages/validators-util/src/isPhoneNr/isPhoneNr.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { cleanup } from "@testing-library/react";
+import { isPhoneNr } from "./isPhoneNr";
+
+describe("isPhoneNr", () => {
+    afterAll(cleanup);
+
+    it("expect to be CORRECT phone number", () => {
+        expect(isPhoneNr("12345678")).toBe(true);
+    });
+
+    it("expect to be INCORRECT phone number", () => {
```

Comment:
Antar dette er mest for eksemples skyld, men vi bør ha en haug mer tester for å verifisere validatorene.

Reply:
Tror også jeg ville omforrmulert litt. F.eks. "returns false for incorrect phone numbers", siden det er det som testes at den gjør. Dette er pirk altså 😅

Reply:
Pirk och inte bara pirk, för att vi må som du säger @lfbergee ha mer tester, validatorer är kritiskt, så mitt förslag, let it be, och optimera, expandera på vägen.